### PR TITLE
e2e: reactivate network test

### PR DESCRIPTION
**Pull request by cmwaters**

### CODE REVIEW

**Review by tychoish — APPROVED** [review]

File: test/e2e/runner/test.go [hunk 1]
```diff
@@ -13,5 +13,5 @@ func Test(testnet *e2e.Testnet) error {
 		return err
 	}
 
-	return execVerbose("./build/tests", "-test.count=1", "-test.v")
+	return execVerbose("./build/tests", "-test.count=1")
```
Comment (williambanfield) [inline test/e2e/runner/test.go:16]: Why this change?

**Review by creachadair — APPROVED** [review]
